spec/support/test_model.rb

```diff
@@ -24,6 +24,9 @@
 
 class User < ActiveRecord::Base
   has_and_belongs_to_many :posts
+  if ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
Example build failure for rails 4+
https://travis-ci.org/sportngin/m2m_fast_insert/builds/151319520